spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/leader/LeaderProperties.java

```diff
@@ -106,7 +106,7 @@ public void setNamespace(String namespace) {
 	}
 
 	public String getNamespace(String defaultValue) {
```

Comment:
We also noticed it... 
Maybe this method should be renamed to `getNamespaceOrDefault` ?